tests/Doctrine/Tests/DBAL/Schema/DB2SchemaManagerTest.php

```diff
@@ -59,5 +61,115 @@ public function testListTableNamesFiltersAssetNamesCorrectly()
             ],
             $this->manager->listTableNames()
         );
+
+        $callable = $this->conn->getConfiguration()->getSchemaAssetsFilter();
+        $this->assertInternalType('callable', $callable);
+
+        // BC check: Test that regexp expression is still preserved & accessible.
```

Comment:
Looks like it's a different concern than filtering assets. Please create a new method instead.